scripts/release.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
I would suggest rename the file to be more specific with the task. 
It is only creating a release branch and not releasing.